tests/FolderTest.php

```diff
@@ -16,7 +16,7 @@ protected function init(): void
         $this->addField('name');
 
         $this->hasMany('SubFolder', [new self(), 'their_field' => 'parent_id'])
-            ->addField('count', ['aggregate' => 'count', 'field' => $this->expr('*')]);
+            ->addField('count', ['aggregate' => 'count', 'field' => $this->expr('*')]); // TODO PHPStan
```

Comment:
`$model->expr()` is not defined, @georgehristov what is a replacement?

Reply:
`$model->persistence->expr()`